distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -253,6 +253,7 @@ public void beforeExam() throws Exception {
             CswSourceProperties cswStubServerProperties =
                     new CswSourceProperties(CSW_STUB_SOURCE_ID);
             cswStubServerProperties.put("cswUrl", CSW_STUB_SERVER_PATH.getUrl());
+            cswStubServerProperties.put(POLL_INTERVAL, CSW_SOURCE_POLL_INTERVAL);
```

Comment:
Yes, the poll interval was changed to 10 minutes to match the other test sources. @figliold noticed that if his itests took long enough that the sources were re-polled, they because marked as unavailable. There should be more digging into why exactly this occurs in itests, but for the time being this should cause the unavailability issues with the cswStubServer to stop. As far as we can tell, this issue does not happen in production, only in the itests.